uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/AdbShellUtils.java

```diff
@@ -66,6 +73,7 @@ public static String executeShellCommandAsCurrentPackage(@NonNull final String c
      * @param packageName the name of the package to install
      */
     public static void installPackage(@NonNull final String packageName) {
+        Logger.i(TAG, "Installs the supplied package on the device..");
```

Comment:
let's log the package in the log message

Reply:
added.